tools/pdf2txt.py

```diff
@@ -5,19 +5,23 @@
 """
 import argparse
 import logging
+logging.basicConfig()
 import six
 import sys
 import pdfminer.settings
 pdfminer.settings.STRICT = False
 import pdfminer.high_level
 import pdfminer.layout
-from pdfminer.image import ImageWriter
 
+OUTPUT_TYPES = ((".htm", "html"),
```

Comment:
I like to keep the git history very clear about why things have changed. This should be removed from this PR, because it is not related to the debug logger mode.

Reply:
Moved to #332